.github/ISSUE_TEMPLATE/page-translation.yml

```diff
@@ -0,0 +1,35 @@
+name: 📄 Page request
+title: "page translation: "
+description: Page translation request template.
+labels: new command, translation
+body:
+  - type: textarea
+    attributes:
+      label: Command description
+      description: Describe a command you want to be translated.
+      placeholder: Tell us what TLDR page you want to see!
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: Command details
+      description: Describe any details related to a command.
+      placeholder: e.g. command version
+  - type: dropdown
+    attributes:
+      label: OS
+      description: Select an operating system category which this command falls into.
+      multiple: false
+      options:
+        - Linux
+        - macOS
+        - Windows
+        - General
+        - Other
+    validations:
+      required: true
+  - type: input
+    attributes:
+      label: OS details
+      description: Describe any details related to an operating system.
+      placeholder: e.g. OS version obtained via `uname -a`
```

Comment:
Only part I'm partial too. I do like that we explain it for Linux users, but this is written like it would work for everyone.

I don't think that would work for Windows/macOS/Android/etc users.

I'd suggest either:
1) Clarify that only works on Linux
2) Remove it

Reply:
I've specified such commands for Linux, macOS, Windows.